platform-api-client-java/core/src/main/java/com/codenvy/client/core/DefaultWorkspaceClient.java

```diff
@@ -33,6 +33,12 @@
  * @author Stéphane Daviet
  */
 public class DefaultWorkspaceClient extends AbstractClient implements WorkspaceClient {
+
+    /* Composite key delimiter.
+     * See {@link WorkspaceManager#getWorkspace(String key)}
```

Comment:
`@link` won't work as long as comment is not javadoc. Use `@see WorkspaceManager#getWorkspace(String key)` instead